### ISSUE TITLE
Incorrect vulnerability details

### ISSUE THREAD

**Issue by TeroPihlaja** [OSSIndex/vulns#171]
**Vulnerability URL**
Provide the URL to the OSS Index vulnerability. eg:
```
https://ossindex.sonatype.org/vulnerability/34a50580-9fa8-499e-803b-8f2f2920a18a
```

**Description**
According to NVD (and even Sonatype description) this has been fixed in new versions:
"The request phase of the OmniAuth Ruby gem (1.9.1 and earlier)"

Looks like the "affected versions" does not reflect this.

**Comment by ken-duck** [comment]
This is a complex problem. Our researchers have determined that the vulnerability has itself never been resolved. The developers released a document on mitigating the problem: https://github.com/omniauth/omniauth/wiki/Resolving-CVE-2015-9284

The problem with this solution is that users who are not aware of the vulnerability will not see the mitigation documents and therefore will remain vulnerable. For that reason we have left the issue in our database.

**Comment by TeroPihlaja** [comment]
I see. It seems that version 2.0+ have a safe default setting that mitigates this issue. This is also noted in the version 2.0 upgrade notes.

I think there is no "right" answer to this discussion.

Should this be closes as "won't fix" then?

**Comment by ken-duck** [comment]
Our researchers concur with your findings, they did fix it in 2.0. We will be fixing our results shortly. Thanks for the help!
